seqr/management/tests/run_postgres_database_backup_2_3_tests.py

```diff
@@ -27,37 +27,36 @@ def test_command_no_argument(self, mock_datetime, mock_os):
             mock.call('DEPLOYMENT_TYPE', 'unknown'),
         ]
         mock_os.environ.get.assert_has_calls(calls)
-        self.assertEqual(
-            '=====================================\n' +
-            '======== 2020-04-27__20-16-01 ======= \n' +
-            '=====================================\n' +
-            'Creating directory: /postgres_backups\n' +
-            '/usr/bin/pg_dump -U postgres --host localhost seqrdb | gzip -c - > /postgres_backups/seqrdb_unknown_backup_2020-04-27__20-16-01.txt.gz\n' +
-            '/usr/local/bin/gsutil mv /postgres_backups/seqrdb_unknown_backup_2020-04-27__20-16-01.txt.gz gs://db_back_bucket/postgres/seqrdb_unknown_backup_2020-04-27__20-16-01.txt.gz\n',
-            out.getvalue())
+        calls = [
+            mock.call('====================================='),
+            mock.call('======== 2020-04-27__20-16-01 ======= '),
+            mock.call('====================================='),
+            mock.call('Creating directory: /postgres_backups'),
+            mock.call('/usr/bin/pg_dump -U postgres --host localhost seqrdb | gzip -c - > /postgres_backups/seqrdb_unknown_backup_2020-04-27__20-16-01.txt.gz'),
+            mock.call('/usr/local/bin/gsutil mv /postgres_backups/seqrdb_unknown_backup_2020-04-27__20-16-01.txt.gz gs://db_back_bucket/postgres/seqrdb_unknown_backup_2020-04-27__20-16-01.txt.gz')
+        ]
+        mock_logger.info.assert_has_calls(calls)
 
         mock_os.system.assert_called_with(
             '/usr/local/bin/gsutil mv /postgres_backups/seqrdb_unknown_backup_2020-04-27__20-16-01.txt.gz gs://db_back_bucket/postgres/seqrdb_unknown_backup_2020-04-27__20-16-01.txt.gz')
 
     # Test the command with different arguments
-    def test_command_with_arguments(self, mock_datetime, mock_os):
-        out = BytesIO()
-
+    def test_command_with_arguments(self, mock_datetime, mock_os, mock_logger):
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/manual/hanars/seqr/pullRequest?prid=5795824)

Reply:
not relevant